### PR TITLE
[TECH-400] Add black cli as a dependency

### PR DESCRIPTION
**Pull request by SamTheisens**
so that `pipenv run format` works from the get-go

branch: feature/TECH-400-install-black-for-dev

----
### 📕 [TECH-400](https://vandebron.atlassian.net/browse/TECH-400) Investigate (and implement) code formatter danielkoves@vandebron.nl 
Investigate the usage of a formatter tool (sort of like eslint/prettier) to keep our code style like newlines, spacing, alignment etc consistent. [](https://black.readthedocs.io/en/latest/) seems like a promising option.

🏗️ Build [3](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-161/3/display/redirect) ✅ Successful, started by _Sam Theisens_  
🚀 *[cloudfront-service](https://cloudfront-service-161.test.nl/)*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-161.test.nl/)*, *[sbtservice](https://sbtservice-161.test.nl/)*, *sparkJob*  

[TECH-400]: https://vandebron.atlassian.net/browse/TECH-400?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+PYTHONPATH=/<absolute_fs_path>/mpyl/src
```
Comment (Jorg88) [inline .env.example:1]: Should we also add any of these? Some might be outdated, you probably know better than me

```
JENKINS_URL=""
JIRA_URL=""
DOCKER_HOST_NAME=""
CHANGE_ID="1"
GITHUB_TOKEN=""
GITHUB_JOB=""
SLACK_TOKEN=""
MPYL_GITHUB_APP_PRIVATE_KEY=""
MPYL_VERSION=""
```

File: tests/reporting/formatting/test_resources/project_console.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+                               Project dockertest                               
+┌─────────────┬────────────────────────────────────────────────────────────────┐
+│ Name        │ dockertest                                                     │
+│ Path        │ test_project.yml                                               │
+│ Description │ This is a test container. For testing the MPL pipelines, not   │
+│             │ to be deployed anywhere.                                       │
+│ Maintainer  │ ['MPyL']                                                       │
+│ Stages      │ Stages(build='Docker Build', test='Docker Test',               │
+│             │ deploy='Kubernetes Deploy', postdeploy='Cypress Test')         │
+└─────────────┴────────────────────────────────────────────────────────────────┘
```
Comment (Jorg88) [inline tests/reporting/formatting/test_resources/project_console.txt:10]: fancy!

**Review by Jorg88 — APPROVED** [review]

File: README-dev.md [hunk 3]
```diff
@@ -72,4 +72,18 @@ which will trigger a build and release to https://pypi.org/project/mpyl/
...
-   which should now succeed.
\ No newline at end of file
+   which should now succeed.
+
+## ..running the mpyl sourcecode against another repository
+
+To test the mpyl sourcecode against the peculiarities of your own repository, you can run the following command:
+
+```shell
+PIPENV_PIPFILE=/Users/samtheisens/IdeaProjects/tmp/mpyl/Pipfile pipenv run cli-ext build status
```
Comment (koeves) [inline README-dev.md:82]:
```suggestion
PIPENV_PIPFILE=</path/to/mpy/repo>/Pipfile pipenv run cli-ext build status
```

File: src/mpyl/cli/projects.py [hunk 4]
```diff
@@ -93,11 +93,25 @@ def shell_complete(self, ctx: click.Context, param, incomplete: str):
...
+@click.pass_context
+def show_project(ctx, name):
+    obj = ctx.obj
+    project_path = f"{name}/{Project.project_yaml_path()}"
+    if not (obj.cli.repo.root_dir() / project_path).exists():
+        obj.cli.console.print(
+            Markdown(
+                f"Project `{name}` not found. 👉 Finding projects is much easier with [auto completion]"
+                f"(https://github.com/Vandebron/mpyl/blob/096df81404985d966b870e7d33697c0101563a88/"
+                f"README-usage.md#auto-completion) enabled."
```
Comment (koeves) [inline src/mpyl/cli/projects.py:105]: This could be the mpyl docs link:
https://vandebron.github.io/mpyl/mpyl.html#mpyl-cli

**Review by koeves — APPROVED** [review]
Looks good, small comment about the links

File: .env.example (hunk 1 above)
Comment (SamTheisens) [reply .env.example:1]: Those other env vars relate to mpyl run configuration. They could also be injected by Jenkins or another task runner execution time. You will also get clear(er) feedback when config values are wrong or missing.
So I don't think there's much point to listing them here as well as in `mpyl_config`

**Pull request merged**